Sources/Quick/World.swift

```diff
@@ -149,11 +149,18 @@ final internal class World: _WorldBase {
         if let group = specs[name] {
             return group
         } else {
-            let group = ExampleGroup(
-                description: "root example group",
-                flags: [:],
-                isInternalRootExampleGroup: true
-            )
+			let group: ExampleGroup
+
+			if let x = specClass as? ResultBuilderQuickSpec.Type {
+				group = x.spec().rootGroup
+			} else {
+				group = ExampleGroup(
+					description: "root example group",
+					flags: [:],
+					isInternalRootExampleGroup: true
+				)
+			}
```

Comment:
The new DSL builds its structure up internally, without needing global state like the `World`.

Still, the `World` is where the examples are ultimately queried from, so I just patched this method for now.